README.md

```diff
@@ -1,4 +1,27 @@
 # What is ActiveRecord ?
 
-1. What is ActiveRecord? 
-2. ActiveRecord vs. SQL
\ No newline at end of file
+## Objectives
+
+1. Define ActiveRecord and understand why it is easier to use.
+2. Compare ActiveRecord vs. SQL queries.
+
+## ActiveRecord
+
+ActiveRecord is what is referred to as an Object-Relational Mapper (ORM). It's responsible for mapping exactly one row in a database to one object in Ruby. 
+
+What this means in practice is that you use ActiveRecord to create or retrieve your data and use traditional Ruby methods and attribute assignments to interact with this data. Instead of composing an SQL query manually, filling in the search parameters, sending it off to a database API, running the query, examining and parsing the result and storing the result in either a hash or a database, you let ActiveRecord do all that for you.
```

Comment:
Going forward, you'll use ActiveRecord to create and retrieve data from your database, and use traditional Ruby methods and attribute assignments to interact with this data. Instead of composing an SQL query manually, filling in the search parameters, sending it off to a database API, running the query, examining and parsing the result and storing the result in either a hash or a database, you let ActiveRecord do all that for you by writing simple Ruby code..